protos/vega/spec_definition.go

```diff
@@ -76,6 +77,20 @@ func (d DataSourceDefinition) GetFilters() []*datapb.Filter {
 					)
 				}
 			}
+		case *DataSourceSpecConfigurationTimeTrigger:
+			if tp != nil {
+				if len(tp.Conditions) > 0 {
+					filters = append(filters,
+						&datapb.Filter{
+							Key: &datapb.PropertyKey{
+								Name: "vegaprotocol.builtin.timetrigger",
```

Comment:
we could probably use the exported const from the timetrigger package hjere.

Reply:
That would create a cycle import because the timetrigger already imports this package as ``vegapb "code.vegaprotocol.io/vega/protos/vega"``